src/scss/abstracts/_variables.scss

```diff
@@ -19,3 +19,12 @@ $gu-grey: $dark-grey;
 
 $accelerator-blue: #009aff;
 $accelerator-yellow: #ffd400;
+
+$font-family-primary: 'Brandon Text';
+$font-family-secondary: 'Brandon Grotesque';
+
+$font-weight-300: 300;
```

Comment:
I really like extracting these values out into variables. However, I'm not 100% on the font weights' names. It doesn't feel like we are gaining anything by saying `$font-weight-500` instead of just `500`. But I also think the same is true if we did versions like `$font-weight-bold`, etc. It'd be nice to use semantic names like, `$font-weight-paragraph` and `$font-weight-heading`, but I also don't know that we actually use font weights like that.

I just wanted to mention it. I don't know that we need to change anything here. It's not like we're changing font-weights en-masse all the time. Or even that there are that many font-weight choices. Just something to consider when naming variables. :D

Reply:
I agree. This process started a little backwards and then I ended up using the font-weight number because it was the only thing that was explicit enough. But I agree, it's redundant and unnecessary. I think I'll leave them for now if that's ok as I don't think it hurts. Happy to remove if you feel like it's over-zealous.